rviz_default_plugins/package.xml

```diff
@@ -64,6 +64,8 @@
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>rviz_rendering_tests</test_depend>
   <test_depend>rviz_visual_testing_framework</test_depend>
+  <test_depend>image_transport_plugins</test_depend>
+  <test_depend>vision_opencv</test_depend>
```

Comment:
Same idea here, `vision_opencv` is not a core package, we can't use this one here